setup.py

```diff
@@ -226,12 +227,14 @@ def run(self):
 extras["modelcreation"] = deps_list("cookiecutter")
 
 extras["serving"] = deps_list("pydantic", "uvicorn", "fastapi", "starlette")
+extras["speech"] = deps_list("datasets", "soundfile")
```

Comment:
Why is datasets needed here? If it's just for testing, then it should go by itself in `testing`. I'd expect `pip install transformers["speech"]` to only install what is relevant to speech.